AbletonMCP_Remote_Script/handlers/devices.py

```
@@ -0,0 +1,1360 @@
+            "macros": macros,
+        }
+    except Exception as e:
+        if ctrl:
+            ctrl.log_message("Error getting macro values: " + str(e))
+        raise
+
+
+def set_macro_value(song, track_index, device_index, macro_index, value, track_type="track", ctrl=None):
+    """Set the value of a specific macro control on a rack device."""
+    try:
+        track, device = _resolve_device(song, track_index, device_index, track_type)
```

Ruff (0.15.0)

[warning] Unpacked variable `track` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)